src/pages/settings/Profile/DisplayNamePage.js

```diff
@@ -74,17 +74,20 @@ function DisplayNamePage(props) {
                 title={props.translate('displayNamePage.headerTitle')}
                 onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_PROFILE)}
             />
-            <Form
+            <FormProvider
                 style={[styles.flexGrow1, styles.ph5]}
                 formID={ONYXKEYS.FORMS.DISPLAY_NAME_FORM}
                 validate={validate}
                 onSubmit={updateDisplayName}
                 submitButtonText={props.translate('common.save')}
                 enabledWhenOffline
+                shouldValidateOnBlur
+                shouldValidateOnChange
             >
                 <Text style={[styles.mb6]}>{props.translate('displayNamePage.isShownOnProfile')}</Text>
                 <View style={styles.mb4}>
-                    <TextInput
+                    <InputWrapper
+                        InputComponent={TextInput}
```

Comment:
As soon as we will introduce TypeScript (migration is in progress) it certainly won't be a problem, because we can create a simple generic that will handle the props properly.